dependency-check-maven-config.xml

```diff
@@ -84,4 +128,103 @@
    ]]></notes>
         <cve>CVE-2014-0003</cve>
     </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        this CVE is generating a lot of false positives it should only include jackson-dataformat-xml jar
+         but it is catching all jackson dependencies unfortunately I can't fix it so if we start depending
+          on a vulnerable version of jackson-dataformat-xml this will sill suppress it.
+   ]]></notes>
+        <cve>CVE-2016-3720</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: commons-beanutils-1.8.3.jar
+   shiro-core has a dependency on this but it doesn’t expose commons-beanutils to user input so it wouldn't pose a risk like the struts library that is called out in the CVE
+   ]]></notes>
+        <sha1>686EF3410BCF4AB8CE7FD0B899E832AABA5FACF7</sha1>
+        <cve>CVE-2014-0114</cve>
+    </suppress>
+
+    <!--todo remove this to ascertain what issues remain -->
+    <suppress>
+        <notes><![CDATA[
+   file name: ffmpeg-2.8.4_2-bin.zip: ffmpeg.exe
+   ]]></notes>
+        <sha1>63C7050695FAD278BC07EDFAC8E4C9C72FA2D754</sha1>
+        <cpe>cpe:/a:ffmpeg:ffmpeg:-</cpe>
+    </suppress>
+
+
+    <suppress>
+        <notes><![CDATA[
+        false positive CVE is unrelated
+   file name: platform-filter-delegate-*.jar
+   ]]></notes>
+        <cve>CVE-2005-0861</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive CVE is unrelated
+   file name: nagasena-0000.0002.0049.0.jar
+   ]]></notes>
+        <sha1>8821DF6D1BC22FBA8F608021F4F27698CF3FD91F</sha1>
+        <cve>CVE-2014-9389</cve>
+    </suppress>
+
+
+    <!-- these are the geowebcache vulnerabilities it is not installed by default and it is only experimental these security issues would need to be resolved before geowebcache can be installed in a production enviroment-->
```

Comment:
Should this comment actually be in a feature description or somewhere visible in the Admin UI?